client/flutter/assets/content_bundles/myth_busters.en.yaml

```diff
@@ -0,0 +1,76 @@
+schema_version: 1
+content_version: 1
+contents:
+  type: mb
```

Comment:
not a blocker for this PR, but we should change these out for more readable types in the future

Reply:
def a blocker for v1 though, good point